fireworks/flask_site/app.py

```diff
@@ -407,4 +406,16 @@ def simple(coll, interval, num_intervals):
 
 
 if __name__ == "__main__":
-    app.run(debug=True, port=8080)
+    # https://dlukes.github.io/flask-wsgi-url-prefix.html
+
+    from fireworks import LaunchPad
+    from werkzeug.middleware.dispatcher import DispatcherMiddleware
+    # from werkzeug.wrappers import Response
```

Comment:
Remove unused code.

Reply:
Generally, I would agree with you but I left there to avoid any confusion. The Response class of Flask used before is a subclass of the Response class of werkzeug. So technically it is ok to use but not necessary to use the flask's version at all and this is why it is in a comment. 
In conclusion, this whole part is just an example of using a non-root URL path. Because it is a "special" case probably it is not even the best place for it :) and just keep:
```python
    from fireworks import LaunchPad

    app.lp = LaunchPad()
    app.run(debug=True, port=8080, threaded=False)
```
or the whole thing could just go into the comment:
```python
    from fireworks import LaunchPad
    
    # https://dlukes.github.io/flask-wsgi-url-prefix.html
    # from werkzeug.middleware.dispatcher import DispatcherMiddleware
    # from werkzeug.wrappers import Response
    #
    # app.wsgi_app = DispatcherMiddleware(
    #     Response('Not Found', status=404),
    #     {'/my-app': app.wsgi_app}
    # )

    app.lp = LaunchPad()
    app.run(debug=True, port=8080, threaded=False)
```

Reply:
Ok, thanks for the information. I'm in favour of your second example. Could you please comment out the whole block and add an additional comment say that this is an example of using a non-robot URL path.

Reply:
@utf Thanks for the feedback! I modified the PR according to your suggestions.